app/celery/process_delivery_status_result_tasks.py

```diff
@@ -231,15 +220,15 @@ def _get_notification_platform_status(self, provider: any, body: str, sqs_messag
         current_app.logger.error("The event stream body could not be translated.")
         current_app.logger.exception(e)
         current_app.logger.debug(sqs_message)
-        self.retry(queue=QueueNames.RETRY)
+        raise AutoRetryException(f'Found {type(e).__name__}, autoretrying...')
 
     current_app.logger.info("retrieved delivery status: %s", notification_platform_status)
 
     # notification_platform_status cannot be None
     if notification_platform_status is None:
         current_app.logger.error("Notification Platform Status cannot be None")
         current_app.logger.debug(body)
-        self.retry(queue=QueueNames.RETRY)
+        raise AutoRetryException(f'Found no notification_platform_status, autoretrying...')
```

Comment:
should this be an f-string?

Reply:
Darn, I went back and double checked after Evan called it out and caught another one but missed this one. Thanks!